test/utils/config.go

```diff
@@ -0,0 +1,135 @@
+package utils
+
+import "testing"
+
+func SetEnvConfig(t *testing.T, snap, service, defaultServicePort string) {
+	if FullConfigTest == "off" {
+		// make this subtest optional to save testing time,
+		// similar full config tests have been operated in device-mqtt test suite
+		t.Skip()
```

Comment:
Add the message inside Skip to explain why it was skipped, i.e. "Full config test is disabled". The message would be shown in test results.